src/Xamarin.Android.Build.Tasks/Tasks/GenerateLayoutBindings.BindingGenerator.cs

```diff
@@ -83,14 +93,12 @@ public void WriteLineIndent (string text)
 				}
 			}
 
+			const string GlobalIdPrefix = "@global";
```

Comment:
What is `@global`? I've never heard of it before, and can't [easily find anything mentioning it](https://www.google.com/search?q=android+resource+id+@global).

Reply:
It's just a name I use to match. And since it's for "global" types I made it "@global" :)